test/spec/modules/jcmBidAdapter_spec.js

```diff
@@ -198,7 +198,7 @@ describe('jcm adapter tests', function () {
     var response = '{"bids":[{"width":300,"cpm":3,"ad":"%3Cimg+src%3D%22http%3A%2F%2Fmedia.adfrontiers.com%2Fimgs%2Fpartnership_300x250.png%22%3E","callbackId":"3c9408cdbf2f68","height":250},{"width":728,"cpm":0,"ad":"%3Cimg+src%3D%22http%3A%2F%2Fmedia.adfrontiers.com%2Fimgs%2Fpartnership_728x90.png%22%3E","callbackId":"3c9408cdbf2f69","height":90}]}';
 
     it('callback function should exist', function () {
-      expect($$PREBID_GLOBAL$$.processJCMResponse).to.exist.and.to.be.a('function');
```

Comment:
Due to [this issue](https://github.com/prebid/Prebid.js/issues/1340#issuecomment-319710240) from the discussion in #1340.

Since their server hardcodes `pbjs` into its JSONP response, we can't change [this line](https://github.com/prebid/Prebid.js/blob/master/modules/jcmBidAdapter.js#L9), without breaking their adapter.

I couldn't change the source, so... I changed the test instead.